swarm/src/connection.rs

```diff
@@ -82,6 +83,12 @@ impl ConnectionId {
     }
 }
 
+impl Display for ConnectionId {
+    fn fmt(&self, f: &mut Formatter) -> fmt::Result {
+        write!(f, "{}", self.0)
+    }
+}
```

Comment:
You'll need to bump the patch version of `libp2p-swarm`, otherwise `libp2p-kad` will not point to the lowest-required version of this crate when we release and won't compile.

Reply:
Ok. I updated the swarm change log as well.